shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -50,6 +55,7 @@ - (instancetype)initWithEngine:(FlutterEngine*)engine
   NSAssert(engine != nil, @"Engine is required");
   self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
   if (self) {
+    _binaryMessenger = [[FlutterBinaryMessengerRelay alloc] initWithParent:self];
```

Comment:
should this be initWithParent:engine.binaryMessenger?

Reply:
No, that would create a relay on top of a relay.  This is how we want it.  Optionally we could remove binaryMessenger from the ViewController header, and/or make it just return the engines binaryMessenger.  I recommend doing neither since it is the easiest migration and the safest change.